swig/Version.i.in

```diff
@@ -3,4 +3,9 @@
 #include <pacbio/Version.h>
 %}
 
+%pythoncode
+%{
+__version__ = "@UNANIMITY_VERSION@"
+%}
+
```

Comment:
let me do this in my fixPython branch, can you remove the move from ConsensusCore.i.in to ConsensusCore.i and this version shift? I'll take care of it there.